components/articles/registration-details.tsx

```diff
@@ -0,0 +1,90 @@
+import {Title} from "../infoColumns/infoColumns.styles";
+import {Body, SubTitle} from "../../styles/typography";
+import {MaterialCard} from "../../app/components/elements";
+import {colors} from "../../styles/colors";
+import styled from "styled-components";
+import Link from "next/link";
+
+
+interface PricingTableProps {}
+
+const PricingTable = styled.table<PricingTableProps>`
+  font-size: 1.5rem;
+  margin-left: 3vw;
+  margin-right: 3vw;
+  border: 3px solid ${colors.indigo};
+`;
+
+const Pricing = () => {
+  return <PricingTable>
```

Comment:
where is the component pricingtable coming from? its not imported like its a component so is it a styled component wrapper? I think the only changes to the table to make it mobile friendly is have breakpoints for the fonts and overall width of the table and maybe add some breakpoints for spacing so when the font size is smaller things aren't so jumbled

Reply:
yeah this comes from the const above it so just a styled component wrapper. I think the table itself is not exactly how I want it to look, but I know nothing about table styling. May want to link up and discuss this soon